ui.py

```diff
@@ -61,35 +60,18 @@ def display_next_question(self):
 
     def display_final_score(self):
         """Display final score"""
-        self.canvas.itemconfig(self.question_text, text=f"""YOU'VE COMPLETED THE QUIZ!\n
-                  Your final score was {self.quiz.score}/{len(self.quiz.question_list)}""")
+        self.canvas.itemconfig(self.question_text, text=f"""YOU'VE COMPLETED THE QUIZ!\nYour final score was 
+{self.quiz.score}/{len(self.quiz.question_list)}""")
         self.window.after(4000, func=self.window.destroy)
 
-    def display_winning(self):
-        """Display an info showing that the player selected the right answer"""
-        self.canvas.itemconfig(self.question_text, text="You're right!")
-        self.score_label.config(text=f"Score: {self.quiz.score}",
-                                fg="white",
-                                bg=THEME_COLOR)
-        self.window.after(2000, func=self.display_next_question)
-
-    def display_losing(self):
-        """Display an info showing that the player selected the wrong answer"""
-        self.canvas.itemconfig(self.question_text, text="You're wrong!")
-        self.window.after(2000, func=self.display_next_question)
-
-    def check_if_true(self):
+    def display_result(self, answer):
         """Check if the True answer is right or wrong"""
-        # THE REASON I DIDN'T MAKE ONE FUNCTION WITH AN ARGUMENT "True" OR "False" IS BECAUSE A TKINTER
-        # REQUIREMENT - A COMMAND FUNCTION FOR A BUTTON MUST NOT CONTAIN ARGUMENTS. OR AM I WRONG? (KH)
-        if self.quiz.check_answer("True"):
-            self.display_winning()
-        else:
-            self.display_losing()
-
-    def check_if_false(self):
-        """Check if the False answer is right or wrong"""
-        if self.quiz.check_answer("False"):
-            self.display_winning()
```

Comment:
konstrukcja tu jest odwrotna - funkcja do wyswietlania odpala validator, to nie jest sprzeczne z separation of concerns